plugin.yaml

```diff
@@ -250,9 +306,9 @@ node_types:
       <<: *playbook_config
     interfaces:
       cloudify.interfaces.lifecycle:
```

Comment:
We should not remove this as it can break existing blueprints and we have not officially deprecated this. How can we modify the code to support the new way while not breaking the old way? Also this is a specific customer request to support this operation in precreate.